asset/css/markbind.css

```diff
@@ -92,3 +92,7 @@ footer {
 #flex-div {
     flex: 1;
 }
+
+keyword {
+    display: none;
+}
```

Comment:
CSS files are usually ordered by increasing specificity (and then alphabetical order).

Reply:
This doesn't fulfil the requirements in the target issue at all. Remove this style.